pkg/chunk/grpc/grpc.proto

```diff
@@ -0,0 +1,121 @@
+syntax = "proto3";
+
+package grpc;
+
+import "google/protobuf/empty.proto";
+
+service grpc_store {
+    //index-client
+    rpc BatchWrite(BatchWrites) returns (google.protobuf.Empty);
+    rpc QueryPages(IndexQuery) returns (stream ReadBatch);
+    rpc Delete(BatchDeletes) returns (google.protobuf.Empty);
+
+    //storage-client
+    rpc PutChunks(PutChunksRequest) returns (google.protobuf.Empty);
+    //In GetChunks rpc request send buf as nil
+    rpc GetChunks(GetChunksRequest) returns (stream GetChunksResponse);
+    rpc DeleteChunks(ChunkID) returns (google.protobuf.Empty);
+
+    //table-client
+    rpc ListTables(google.protobuf.Empty) returns (ListTablesResponse);
+    rpc CreateTable(TableDesc) returns (google.protobuf.Empty);
+    rpc DeleteTable(TableName) returns (google.protobuf.Empty);
+    rpc DescribeTable(TableName) returns (DescribeTableResponse);
+    rpc UpdateTable(UpdateTableRequest) returns (google.protobuf.Empty);
+
+    //used by storage-client & index-client
+    rpc Stop(google.protobuf.Empty) returns (google.protobuf.Empty);
```

Comment:
So, Initially I was thinking there is no use of Stop(). But I was thinking in next iteration when we add support for multiple grpc-stores. If for some reason we want to terminate the connection from the client-side with one of the grpc-store maybe this can help, just a thought. We can definitely drop this. If we have no use.

Reply:
I would suggest to remove `Stop()` and add it if/when required.